src/test/java/uk/gov/pay/api/it/PaymentResourceSearchITest.java

```diff
@@ -599,7 +601,30 @@ public void searchPayments_ShouldIncludeCaptureLink_whenReturnedFromConnector()
                 .body("results[0]._links", hasKey("capture"))
                 .body("results[0]._links.capture.method", is("POST"))
                 .body("results[0]._links.capture.href", is("http://publicapi.url/v1/payments/" + chargeId + "/capture"));
+    }
+    
+    @Test
+    public void searchPayments_getsResultsFromConnector_withNoBillingAddress() {
```

Comment:
Looks we don’t have a test for if there is a billing address asserting that one is there — might be useful to add one for completeness

Reply:
This is already covered by `searchPayments_shouldOnlyReturnAllowedProperties`

Reply:
Right you are!